### PR TITLE
skel.ebuild: add global PATCHES array

### PR DESCRIPTION
**Pull request by juippis**

### CODE REVIEW

File: skel.ebuild [hunk 1]
```diff
@@ -105,6 +105,14 @@ IUSE="gnome X"
 #BDEPEND="virtual/pkgconfig"
 
 
+# Optional patches to make the program compile properly. Gets applied in the
+# src_prepare() phase with EAPIs >=6. Can hold more than one patch in an array
+# form. Patches are placed in the package's files/ directory, e.g.
+# dev-libs/openssl/files/openssl--1.2.3-respect-cflags.patch
```
Comment (juippis) [inline skel.ebuild:111]:
```suggestion
# dev-libs/openssl/files/openssl-1.2.3-respect-cflags.patch
```

File: skel.ebuild [hunk 2]
```diff
@@ -105,6 +105,14 @@ IUSE="gnome X"
...
+# Can also be used in src_prepare() with "local PATCHES=( ... )"
+#PATCHES=( "${FILESDIR}"/${PN}-1.2.3-respect-cflags.patch )
```
[the remark this answers is not in the document]
Comment (ionenwks) [reply skel.ebuild:113]: I guess using ${P} over ${PN} could also be debated (to re-evaluate the patch on bumps), but while I like it, I know some devs really hate this too.

[the remark this answers is not in the document]
Comment (mgorny) [reply skel.ebuild:113]: Fun stuff: I actually tend to use `${P}` when I add a patch that's upstreamed or has a good chance of being merged upstream (i.e. to provoke patch failure on bump) and explicit version when I know I'm going to end up maintaining it for a long time.

[the remark this answers is not in the document]
Comment (juippis) [reply skel.ebuild:113]: https://gitweb.gentoo.org/repo/gentoo.git/tree/app-containers/lxd/lxd-4.0.8-r1.ebuild#n75
```
PATCHES=( "${FILESDIR}"/lxd-4.0.8-btrfs-progs-5.14-safety-check-fix.patch
	"${FILESDIR}"/lxd-4.0.7-no-werror.patch )
```
I just plaintext it...

**Comment by juippis** [comment]
So we agreed not to include optional variables.

**Pull request closed without merging**